jenkins/databricks/build.sh

```diff
@@ -163,7 +163,8 @@ fi
 if [[ "$WITH_BLOOP" == "1" ]]; then
     MVN_OPT="-DbloopInstall $MVN_OPT"
     MVN_PHASES="clean install"
-    export JAVA_HOME="/usr/lib/jvm/zulu11"
+    export JAVA_HOME="/usr/lib/jvm/zulu17"
```

Comment:
this is ok for all supported Databricks versions?

Reply:
good point it is not available on DBR 11.3